### ISSUE TITLE
Rohan Taylor getting 668 fewer points.

### ISSUE THREAD

**Issue by philderbeast** [BlockScope/flare-timing#255]
For Dalby 2017 task 4 flare-timing is scoring Rohan Taylor 93 whereas FS scores him 761 points.

**Comment by philderbeast** [comment]
Rohan doesn't make the turnpoint WARRA, even with the give as set.

<img width="478" alt="Screen Shot 2019-05-03 at 10 08 41 AM" src="https://user-images.githubusercontent.com/633283/57142826-7f971400-6d8b-11e9-8ff0-69bd4c5e9723.png">

**Comment by philderbeast** [comment]
Here are the tracks of Tony Giammichele and Rohan Taylor exiting the start cylinder. The markers show the fixes crossing the zone and the interpolated point. There's no such markers on Rohan's track.

<img width="338" alt="Screen Shot 2019-05-03 at 10 17 36 AM" src="https://user-images.githubusercontent.com/633283/57143385-c1748a00-6d8c-11e9-9e53-c7eca8e03d9c.png">

**Comment by philderbeast** [comment]
It looks like there's a glitch because of Rohan not making WARRA. His final glide entry into the start cylinder is being selected as his start as can be seen by the crossing and tag markers on his track.

<img width="675" alt="Screen Shot 2019-05-03 at 10 45 45 AM" src="https://user-images.githubusercontent.com/633283/57145186-ac99f580-6d90-11e9-9e88-d409a53f6fe8.png">

**Comment by philderbeast** [comment]
Here's his crossing for this task:

```yaml
flying:
- []
- []
- []
- - - - '9'
      - Rohan Taylor
    - loggedFixes: 3142
      flyingFixes:
      - 0
      - 3141
      scoredFixes:
      - 0
      - 3141
      loggedSeconds: 13037
      flyingSeconds:
      - 0
      - 13037
      scoredSeconds:
      - 0
      - 13037
      loggedTimes:
      - 2017-04-14T02:33:22Z
      - 2017-04-14T06:10:39Z
      flyingTimes:
      - 2017-04-14T02:33:22Z
      - 2017-04-14T06:10:39Z
      scoredTimes:
      - 2017-04-14T02:33:22Z
      - 2017-04-14T06:10:39Z
- []
crossing:
- []
- []
- []
- - - - '9'
      - Rohan Taylor
    - zonesCrossSelected:
      - crossingPair:
        - fix: 3040
          time: 2017-04-14T06:03:38Z
          lat: -27.1327
          lng: 151.30516667
        - fix: 3041
          time: 2017-04-14T06:03:42Z
          lat: -27.13303333
          lng: 151.30451667
        inZone:
        - false
        - true
      - null
      - null
      - null
      zonesCrossNominees:
      - - crossingPair:
          - fix: 409
            time: 2017-04-14T03:01:29Z
            lat: -27.16083333
            lng: 151.21393333
          - fix: 410
            time: 2017-04-14T03:01:33Z
            lat: -27.16063333
            lng: 151.21313333
          inZone:
          - true
          - false
        - crossingPair:
          - fix: 3040
            time: 2017-04-14T06:03:38Z
            lat: -27.1327
            lng: 151.30516667
          - fix: 3041
            time: 2017-04-14T06:03:42Z
            lat: -27.13303333
            lng: 151.30451667
          inZone:
          - false
          - true
      - []
      - - crossingPair:
          - fix: 2368
            time: 2017-04-14T05:17:23Z
            lat: -26.94013333
            lng: 151.4338
          - fix: 2369
            time: 2017-04-14T05:17:26Z
            lat: -26.93998333
            lng: 151.43446667
          inZone:
          - false
          - true
        - crossingPair:
          - fix: 2373
            time: 2017-04-14T05:17:42Z
            lat: -26.9413
            lng: 151.43658333
          - fix: 2374
            time: 2017-04-14T05:17:46Z
            lat: -26.94188333
            lng: 151.4361
          inZone:
          - true
          - false
      - - crossingPair:
          - fix: 14
            time: 2017-04-14T02:34:18Z
            lat: -27.16095
            lng: 151.25395
          - fix: 15
            time: 2017-04-14T02:34:22Z
            lat: -27.16076667
            lng: 151.2533
          inZone:
          - true
          - false
        - crossingPair:
          - fix: 35
            time: 2017-04-14T02:35:44Z
            lat: -27.15151667
            lng: 151.25725
          - fix: 36
            time: 2017-04-14T02:35:48Z
            lat: -27.1517
            lng: 151.25801667
          inZone:
          - false
          - true
        - crossingPair:
          - fix: 64
            time: 2017-04-14T02:37:43Z
            lat: -27.16478333
            lng: 151.2562
          - fix: 65
            time: 2017-04-14T02:37:47Z
            lat: -27.16481667
            lng: 151.2556
          inZone:
          - true
          - false
        - crossingPair:
          - fix: 239
            time: 2017-04-14T02:49:46Z
            lat: -27.1515
            lng: 151.25765
          - fix: 240
            time: 2017-04-14T02:49:50Z
            lat: -27.15191667
            lng: 151.25736667
          inZone:
          - false
          - true
        - crossingPair:
          - fix: 254
            time: 2017-04-14T02:50:46Z
            lat: -27.15906667
            lng: 151.25388333
          - fix: 255
            time: 2017-04-14T02:50:50Z
            lat: -27.15966667
            lng: 151.25363333
          inZone:
          - true
          - false
        - crossingPair:
          - fix: 299
            time: 2017-04-14T02:53:52Z
            lat: -27.16491667
            lng: 151.2563
          - fix: 300
            time: 2017-04-14T02:53:56Z
            lat: -27.16511667
            lng: 151.25685
          inZone:
          - false
          - true
        - crossingPair:
          - fix: 300
            time: 2017-04-14T02:53:56Z
            lat: -27.16511667
            lng: 151.25685
          - fix: 301
            time: 2017-04-14T02:54:00Z
            lat: -27.16553333
            lng: 151.25668333
          inZone:
          - true
          - false
        - crossingPair:
          - fix: 303
            time: 2017-04-14T02:54:08Z
            lat: -27.16478333
            lng: 151.25605
          - fix: 304
            time: 2017-04-14T02:54:12Z
            lat: -27.16421667
            lng: 151.25631667
          inZone:
          - false
          - true
        - crossingPair:
          - fix: 308
            time: 2017-04-14T02:54:29Z
            lat: -27.1645
            lng: 151.25628333
          - fix: 309
            time: 2017-04-14T02:54:34Z
            lat: -27.16438333
            lng: 151.25575
          inZone:
          - true
          - false
        - crossingPair:
          - fix: 3091
            time: 2017-04-14T06:07:06Z
            lat: -27.1547
            lng: 151.2733
          - fix: 3092
            time: 2017-04-14T06:07:09Z
            lat: -27.15501667
            lng: 151.27276667
          inZone:
          - false
          - true
- []
```

**Comment by philderbeast** [comment]
Now that I'm only selecting crossings for the start that are exits I see that there's still a difference in distance with flare-timing giving Rohan reach up until WARRA of `36.3 km` and FS giving him the whole task, `114.4 km`.

FS has him tagging WARRA even though it also reports that he didn't cross this turnpoint.

![rohan-missed-warra](https://user-images.githubusercontent.com/633283/57179756-34891800-6ed5-11e9-99ed-e51886879435.png)

**Comment by philderbeast** [comment]
I changed the `extract-input` options:
```bash
++ --give-fraction=0.005 --haversines
-- --give-fraction=0.0005 --give-distance=5 --haversines
```

This increased the WARRA turnpoint radius, the give, from `2005 m` to `2010 m`. It looks like Rohan should now be tagging WARRA but flare-timing is not showing this:

<img width="303" alt="Screen Shot 2019-05-04 at 10 00 00 AM" src="https://user-images.githubusercontent.com/633283/57180180-22b36080-6e54-11e9-87ca-8bfe58107c5d.png">

**Comment by philderbeast** [comment]
After the fix for #328, Rohan is getting exactly the same points for this task as FS awards him.
